### ISSUE TITLE
Update Action Management Player Integration Documentation 

### ISSUE THREAD

**Issue by afranzese99** [uchicago-cs/chiventure#835]
The [action management player integration documentation](https://github.com/uchicago-cs/chiventure/wiki/Action-Management-Player-Integration-Plan) describes the code and team needs to integrate the player struct into action management. 

We need to go through the existing code and ensure the descriptions in the documentation properly reflect the current state of the code. Issue #826 will also need to be completed before the documentation is entirely up to date.

**Comment by afranzese99** [comment]
There was outdated code present in the Action Management Player Integration Plan documentation, so the code discussed in the documentation was updated to properly represent its current state. The descriptions and blurbs were also updated to reflect the changes in the code. The next step (represented by issue #826 ) will be to meet with the RPG stats, skill trees, and class teams (and any other relevant teams that may be identified) to discuss their current desires for the player struct and to update the documentation as necessary.

**Comment by MaxineK36** [comment]
Issue Score: ✔️++

Comments:
Great work! I appreciate that you linked to the follow-up issue in your closing statement.
